### ISSUE TITLE
Can you upload some essential files.

### ISSUE THREAD

**Issue by xuhuaren** [lfz/DSB2017#13]
Thanks for your code, good work. While, I'm a little confused about the config files.

config = {'stage1_data_path':'/work/DataBowl3/stage1/stage1/', # Kaggle第一阶段原始数据
          'luna_raw':'/work/DataBowl3/luna/raw/', #LUNA16原始数据
          'luna_segment':'/work/DataBowl3/luna/seg-lungs-LUNA16/', #LUNA16肺分割mask
          'luna_data':'/work/DataBowl3/luna/allset', #LUNA16预处理之后的数据
          'preprocess_result_path':'/work/DataBowl3/stage1/preprocess/', # Kaggele第一阶段预处理之后的数据           
          'luna_abbr':'./detector/labels/shorter.csv', # LUNA16的文件名对数据之间的映射（原始文件名为1.3.6.1.4.1.14519.5.2.1.6279.6001.100225287222365663678666836860，之后用0表示）
          'luna_label':'./detector/labels/lunaqualified.csv', # 筛选之后的LUNA16的结节信息（排除官方给出的结节中直径较小的那些）
          'stage1_annos_path':['./detector/labels/label_job5.csv',
                './detector/labels/label_job4_2.csv',
                './detector/labels/label_job4_1.csv',
                './detector/labels/label_job0.csv',
                './detector/labels/label_qualified.csv'],
          'bbox_path':'../detector/results/res18/bbox/',
          'preprocessing_backend':'python'
         }

can you give me the luna_abbr, luna_label files. If have time, can you explain each items in this config files.

**Comment by JohnnyGambler** [comment]
please read the code carefully ! you can find all config files in the right directory .

**Comment by xuhuaren** [comment]
@JohnnyGambler thanks for your work!

**Comment by liueagle** [comment]
Can you tell me which is a training set, and which is a test set?

**Comment by BilitisWithDing** [comment]
@xuhuaren But I don't find any about getting luna segment code in prepare.py of training directory.can you tell me where is the path of luna_segment?

**Comment by BilitisWithDing** [comment]
@xuhuaren ut I don't find any about getting luna segment code in prepare.py of training directory. Can you tell me where  the path of luna_segment is?
